Formula/tunnel.rb

```diff
@@ -19,15 +19,7 @@ def install
   end
 
   test do
-    begin
-      pid = fork do
-        $stdout.reopen("#{testpath}/out", "w")
-        exec bin/"tunnel", "8080"
-      end
-      sleep 5
-      assert_match "labstack.me", (testpath/"out").read
-    ensure
-      Process.kill("HUP", pid)
-    end
+    exec bin/"tunnel", "start", "8080"
```

Comment:
The following line will never get executed if you do not fork but `exec` directly in this case.

Reply:
This command itself forks within and comes out.

Reply:
As long as you call `exec`, the `ruby` process will be replaced by `tunnel`. Even though `tunnel` does `fork` another process and then “comes out”, that does not mean the process will return back to `ruby` and continue executing. Please see the `man` page for `exec(3)`, or the ruby doc for `Kernel.exec`.

Reply:
I know how `exec` works and if I use fork, I have to put sleep and wait for tunnel to get started. This has been already test.

Reply:
I think `exec` without `fork`ing inside the `test do` block is like some sort of hijacking the `brew test` command. I'd like to let other maintainers to take a look at this. cc @MikeMcQuaid @Homebrew/maintainers

Reply:
Sure, but like I said it is not blocking and passes the test.

Reply:
Yes, please re-add the fork, thanks.